packages/core/src/utils/fragments.ts

```diff
@@ -42,11 +44,12 @@ export function buildFactoryFragments({
   const fragments = buildFragments({
     address,
     topics,
+    includeTransactionReceipts,
     chainId,
     childAddressLocation,
     eventSelector,
     idCallback: (address_, topic0_, topic1_, topic2_, topic3_) =>
-      `${chainId}_${address_}_${eventSelector}_${childAddressLocation}_${topic0_}_${topic1_}_${topic2_}_${topic3_}`,
+      `${chainId}_${address_}_${eventSelector}_${childAddressLocation}_${topic0_}_${topic1_}_${topic2_}_${topic3_}_${includeTransactionReceipts}`,
```

Comment:
I think there's a bug in this file somewhere that is causing the fragment IDs to sometimes use `_false` instead of `_0`.